EIPS/eip-4762.md

```diff
@@ -268,6 +254,13 @@ class AccessSubtree(Container):
     elements: BitVector[256]
 ```
 
+## Block-level operations
+
+* Precompile accounts, system contract accounts and slots of a system contract that are accessed during a system call are considered warm.
```

Comment:
should we add that the code chunks during system calls are not to be warmed? or it might cause additional confusion?

Reply:
system calls totally ignore code chunks imo both for accesses and gas compute and we can clearly mention it to remove confusion